presto-docs/src/main/sphinx/functions/regexp.rst

```diff
@@ -82,6 +88,27 @@ with a few notable exceptions:
 
         SELECT regexp_like('1a 2b 14m', '\d+b'); -- true
 
+.. function:: regexp_position(string, pattern) -> integer
+
+    Returns the index of the first occurrence of ``pattern`` in ``string``.
+    Returns -1 if not found, otherwise returns index counting from 1.
+    ``pattern`` should not be an empty string.::
+
+        SELECT regexp_position('9102, say good bye', '\s*[a-z]+\s*'); -- 6
+
+.. function:: regexp_position(string, pattern, start) -> integer
+
+    Returns the index of the first occurrence of ``pattern`` in ``string`` from ``start``.
+    The value of ``start`` starts from 1::
+
+        SELECT regexp_position('natasha, 9102, miss you', '\s*[a-z]+\s*', 9); -- 15
+
+.. function:: regexp_position(string, pattern, start, occurrence) -> integer
+
+    Returns the index of the ``occurrence``-th occurrence of ``pattern`` in ``string`` from ``start``::
```

Comment:
I think we should repeat for these function what happens when match is not found